src/CoreBundle/Resources/views/snippets-cms/TCMSFieldWYSIWYG/cKEditor/editor.html.twig

```diff
@@ -1,4 +1,11 @@
-<div>
+<div
```

Comment:
For now this is fine.

If we apply the versioning feature to other field types later, we should consider moving the information these data attributes keep to somewhere else. Maybe we can do something more generic so that we a) do not have to adjust all field views, b) have the specific code encapsulated in the ChangeLogBundle and c) can use the field information for other features later.

Reply:
Looked through this once again, there are surely some proper ways of supporting all fields while establishing a consistent ui for the version selection. The ticket was only meant to introduce this feature for html editor fields and all extensions can be picked up separately.